Add Bulk Leave Group DM (#610)
discord-userdoccers/discord-userdoccers: pages/resources/channel.mdx

| ------- | ------- | ------------------------------------------------------------------------------ |
| silent? | boolean | Whether to leave the group DM without sending a system message (default false) |

<RouteHeader method="POST" url="/users/@me/channels/bulk-leave">
  Bulk Leave Group DMs 
</RouteHeader>

> Leave multiple group DMs. Returns a 204 empty response on success. Fires multiple [Channel Delete](/gateway/gateway-events#channel-delete) gateway events.

Reviewer: ```suggestion
Leave multiple group DMs. Returns a 204 empty response on success. Fires multiple [Channel Delete](/gateway/gateway-events#channel-delete) Gateway events.
```

| ------- | ------- | ------------------------------------------------------------------------------ |
| silent? | boolean | Whether to leave the group DM without sending a system message (default false) |

<RouteHeader method="POST" url="/users/@me/channels/bulk-leave">
  Bulk Leave Group DMs 
</RouteHeader>

Leave multiple group DMs. Returns a 204 empty response on success. Fires multiple [Channel Delete](/gateway/gateway-events#channel-delete) gateway events.

> Any channels IDs given that do not exist or are invalid will count towards the minimum and maximum channel count (currently 1 and 1000 respectively).

Reviewer 2: (only not marking as requesting changes because I want others' opinions on wording)
```suggestion
<Alert type="warn">
Any channels IDs given that do not exist or are invalid **will** count towards the limit.
</Alert>
```

Reviewer: i don't think this is worth mentioning at all, it's pretty standard for size validation to happen before existence validation

Author: @dolfies Curious, but shouldn't the same-ish description be removed from [Bulk Delete Messages](https://docs.discord.food/resources/message#bulk-delete-messages) as well for consistency? 

> Any message IDs given that do not exist or are invalid will count towards the minimum and maximum message count (currently 2 and 100 respectively).

Reviewer: yeah i'd say so, that hasn't been rly touched from ddevs docs i think

> Any channels IDs given that do not exist or are invalid will count towards the minimum and maximum channel count (currently 1 and 1000 respectively).

Reviewer 2: ```suggestion
```
